docs/setup-guide/run-e2e-test-locally.md

```diff
@@ -0,0 +1,64 @@
+# How to run e2e test in local dev environment
+
+This doc is to describe how to run e2e test in your local dev environment. Please note, local dev environment is not fitting for many existing e2e test cases. So if those tests failed, it's not a surprise.
+
+1 Install kubetest
+
+E2e test is launched by kubetest. Please visit [kubetest](https://github.com/kubernetes/test-infra/blob/master/kubetest/README.md) to understand it. There is the latest info how to install kubetest if the script in this doc is not working.
+
+(This doc suggests and assumes arktos is installed under $GOPATH/src/k8s.io/)
+
+```bash
+cd $GOPATH/src/k8s.io
+git clone https://github.com/kubernetes/test-infra.git
+cd $GOPATH/src/k8s.io/test-infra/
+GO111MODULE=on go install ./kubetest
+```
+
+kubetest will be installed in $GOPATH/bin/. You may want to add $GOPATH/bin/ to $PATH for easily invoking binaries under the path.
+
+2 	kubetest has a limitation: it only runs under directory of Kubernetes.
+   	You can rename arktos directory to kubernetes.
+   Another way which may be better, is to create a soft link to arktos:
+   
+```bash
+cd $GOPATH/src/k8s.io
+ln -s ./arktos kubernetes
+```
+
+3 Start local cluster
+
+```bash
+cd $GOPATH/src/k8s.io/kubernetes
+make quick-release
```

Comment:
arktos-up does not rely on "make quick-release", is make quick-release necessary for e2e test?

Reply:
Per discussion, please add note why make quick-release is necessary

Reply:
Added